docs/guide/en/test-framework.md

```diff
@@ -0,0 +1,47 @@
+# Running framework tests
+
+Get the source and place it into a host-volume folder for mounting it into the container.
+
+```bash
+git clone https://github.com/yiisoft/yii-core _host-volumes/yii-core
```

Comment:
Why it's there? yii-core is no longer valid.

Reply:
Looks like it's the doc that was transferred from root dir w/o any change.

Reply:
because it and the other files were transferred from the root to the 'en' folder, as now there is the 'pt-BR' folder too. And who knows, maybe one day someone will translate it into another language.
PLEASE see this file in full, as it seems to me that it only refers to "/yii-core" which, as you said, is no longer used. In that case, why keep it?